firmware/mcal/cli/periph/can.h

```diff
@@ -26,18 +27,13 @@ class CanBase : public shared::periph::CanBase {
     }
 
     void Send(const shared::can::RawCanMsg& can_tx_msg) {
-        std::cout << iface_ << " [" << std::hex << std::uppercase
-                  << std::setfill('0') << std::setw(2) << can_tx_msg.header.id
-                  << "] ";
+        "[ "<< std::format("{} {:02X}", iface_ can_tx.msg.header.id) << "] ";
```

Comment:
Where does this get printed? Why aren't the square brackets in the format string? Also I don't think the format string is equivalent to the old statement